src/main/java/pl/allegro/tech/build/axion/release/domain/scm/ScmPosition.java

```diff
@@ -93,7 +89,19 @@ public boolean getIsReleaseBranch() {
     }
 
     @Input
-    public boolean getIsHeadOnVersionTagCommit() {
-        return isHeadOnVersionTagCommit;
+    public boolean isSnapshot() {
```

Comment:
To be consistent with `getIsReleaseBranch`:
```suggestion
    public boolean getIsSnapshot() {
```